api/filter/http/http_connection_manager.proto

```diff
@@ -73,27 +102,38 @@ message HttpConnectionManager {
   // processed sequentially as request events happen.
   repeated HttpFilter http_filters = 5;
 
-  // Whether the connection manager manipulates the user-agent and
-  // x-envoy-downstream-service-cluster headers. See the linked documentation
-  // for more information. Defaults to false.
+  // Whether the connection manager manipulates the *user-agent* and
+  // *x-envoy-downstream-service-cluster* headers. See the linked documentation
```

Comment:
Can you start a "docs contributor guide" somewhere and just fill in for now some of these stylistic conventions you're using here? I have been using inline code quotes and also no formatting for some of these, so it would be good to make the conventions explicit.

Reply:
Yes I will add a contrib guide to this PR just to get it done.